### PR TITLE
Blogpost: DHCP subnet

### PR DESCRIPTION
**Pull request by lzap**
Setting up VMs on my Ryzen workstation, might be helpful to users. Also this does not work for me:

  rake new_post title="XYZ"

I had to modify it. Not sure why it does not work, not Rake expert.

### CODE REVIEW

File: _posts/2017-07-18-adding-new-subnet-for-provisioning.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+And that's pretty much it! I am creating new host, selecting the domain, the
+subnet, IP reservation is populated and entry is submitted. Before I try to
+discover host, I will make sure that Discovery Smart Proxy feature is
+installed on any Smart Proxy accessible from my new Subnet (that would be
+typically the very same host).
+
+Oh, I almost forgot! I don't want the foreman-installer to overwrite my
+customized dhcpd.conf file anymore, therefore I will tell it not to do so:
+
+    # grep dhcp: /etc/foreman-installer/scenarios.d/satellite-answers.yaml
```
Comment (GregSutcliffe) [inline _posts/2017-07-18-adding-new-subnet-for-provisioning.md]: should be `foreman-answers.yaml` right? ;)

Comment (lzap) [reply _posts/2017-07-18-adding-new-subnet-for-provisioning.md]: Yeah, I just happen to have a Sat VM up. No wrong grammar in text? That cannot be true.

Comment (GregSutcliffe) [reply _posts/2017-07-18-adding-new-subnet-for-provisioning.md]: Actually yes, it's all fine apart from that typo! There's nothing really *wrong* about the text - maybe stuff I would write differently, but that's just differences of writing style :)

Good to merge if you could just fix that.

Comment (lzap) [reply _posts/2017-07-18-adding-new-subnet-for-provisioning.md]: Done, added once sentence about TFTP, I just learned that it overwrites grub2/default file if you change it from Foreman.

**Comment by GregSutcliffe** [comment]
Thanks @lzap - I'm going to strip the Rake part of the commit, since it's not needed - if you check the readme, you'll see you call

```
rake new_post['XYZ']
```

instead of 

```
rake new_post title='XYZ'
```

So, it's not broken. If you think the way we specify the title should change that's fine, but lets do a separate commit for that.

**Comment by GregSutcliffe** [comment]
Merged as 2c31e523f, ty @lzap!

**Comment by lzap** [comment]
How often does it generate? I don't see it: https://theforeman.org/blog/index.html

**Comment by mmoll** [comment]
i do 😖

**Pull request closed without merging**